src/Proto3/Suite/DotProto/TH.hs

```diff
@@ -0,0 +1,33 @@
+
+module Proto3.Suite.DotProto.TH (
+  proto3,
+) where
+
+import "template-haskell" Language.Haskell.TH as TH
+import "template-haskell" Language.Haskell.TH.Syntax as TH
+import "template-haskell" Language.Haskell.TH.Quote (QuasiQuoter (..))
+
+import Proto3.Suite.DotProto.AST (Path (..))
+import Proto3.Suite.DotProto.Parsing (parseProtoWithFile)
+
+--------------------------------------------------------------------------------
+
+proto3 :: QuasiQuoter
+proto3 = 
```

Comment:
This name seems a little short.  There are various things we could splice in, and in this case it is the AST we are splicing in, not generated encoders and decoders.  If the name could reflect that somehow, then if we later add another QuasiQuoter for another purpose, then there would be less confusion.

Reply:
Maybe there is a better name, that seems reasonable and I can't think of anything better.